war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -166,6 +166,19 @@ var createPluginSetupWizard = function(appendTarget) {
 			title: text
 		}).tooltip('show');
 	});
+	
+	// handle clicking links that might not get highlighted due to position on the page
+	$wizard.on('click', '.nav>li>a', function(){
+		var $li = $(this).parent();
+		var activateClicked = function() {
+			if(!$li.is('.active')) {
+				$li.parent().find('>li').removeClass('active');
+				$li.addClass('active');
+			}
+		};
+		setTimeout(activateClicked, 150); // this is the scroll time
+		setTimeout(activateClicked, 250); // this should combat timing issues
```

Comment:
Well, yes. Unfortunately, there's no (documented) way to hook in to the bootstrap scrollspy with a 'scroll end' event or anything, so I have to work around whatever it decides to do.